### PR TITLE
Chapter 1 HW3

### PR DESCRIPTION
**Pull request by daman92**
Everything works, but I think I messed up the checks part. Any suggestions are appreciated.

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+		public static void Demo()
+		{
+			/*2) Take homework 2 code, place it in the forked branch and refactor it using functions.There should be as little duplicate code as possible (there should be functions for:
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Remove such comments from code.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+			-Prompt for input and converting it to float(print message for request, read console input and return converted input to float).
+			3) Put all the function you made into the right places in Checks.cs class. Run tests, make sure you pass all the tests
+			4) Put all program.cs logic to Lesson3.cs.Call it from main function. Program class should look like this:
+			*/
+
+			string quit = "";
+
+			while (quit != "q" && quit != "Q")
+			{
+				int age = getUserInt("age");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Use a function that tells what is being done. You need to process person info, might as well call the contents of the loop like that.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+				quit = Console.ReadLine();
+			}
+
+		}
+
+		public static float calculateBMI(float weight, float height)
+		{
+			float heightInMeters = height / 100f;
+			return (weight / (heightInMeters * heightInMeters));
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:27]: No need a blank line before ending a function.

Comment (Almantask) [reply Src/BootCamp.Chapter/Lesson3.cs:27]: It's actually slightly more clean putting a blank line before return.

**Review by Almantask — CHANGES_REQUESTED** [review]
Be more careful about how you do things, because this is more than is asked and therefore the tests are failing. You can still make extra effort and make it pass with the same idea that you have now, but don't- you ain't gonna need it.

A+ for effort though 😄

**Comment by Almantask** [comment]
> Everything works, but I think I messed up the checks part. Any suggestions are appreciated.

Tests are failing, because you did more than is asked. In a tests, for this homework, no validation is needed and you just pretend to do a happy path.

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 2 above)
Comment (daman92) [reply Src/BootCamp.Chapter/Lesson3.cs]: refactored per instruction.

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 1 above)
Comment (daman92) [reply Src/BootCamp.Chapter/Lesson3.cs]: removed per instructions. (sorry that was just for me.)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 3 above)
Comment (daman92) [reply Src/BootCamp.Chapter/Lesson3.cs:27]: made formatting uniform, per comment.

**Comment by Almantask** [comment]
Looks good

**Pull request closed without merging**